data/src/main/java/com/globant/data/database/PokemonDao.kt

```diff
@@ -4,13 +4,20 @@ import androidx.room.Dao
 import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
-import com.globant.data.database.entity.PokemonTypeRoom
+import com.globant.data.database.entity.PokemonSupertypeDatabaseEntity
+import com.globant.data.database.entity.PokemonTypeDatabaseEntity
 
 @Dao
 interface PokemonDao {
     @Query("SELECT * FROM pokemon_type_table")
-    fun getPokemonTypes(): List<PokemonTypeRoom>
+    fun getPokemonTypes(): List<PokemonTypeDatabaseEntity>
 
     @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
this strategy is not the correct, should be REPLACE. In this way, you will always save the last info

Reply:
refactored, thanks